corehq/apps/sms/handlers/form_session.py

```diff
@@ -58,7 +58,7 @@ def form_session_handler(v, text, msg):
                 xforms_session_couch_id=session._id,
             )
             add_msg_tags(msg, inbound_metadata)
-
+            msg.save()
```

Comment:
Prior to adding this line, SMS DB records for incoming messages had `None` value for xforms_session_couch_id and other meta attributes. At least that's what _testMultimediaSubmission_ test revealed. However I don't remember having issues while I was testing on staging and my local, and not having meta attributes would definitely produce some bugs. So I am not sure whether this line is actually necessary but without it _testMultimediaSubmission_ will fail (submitted form doesn't have any attachments)